trunk/SUAVE/Optimization/Package_Setups/scipy_setup.py

```diff
@@ -97,3 +107,185 @@ def SciPy_Problem(problem,x):
 
     
     return obj
+
```

Comment:
Does a regression exist that uses this? Can we tack it onto another optimization?

Reply:
regression added - I used some of @timdmacdo 's methodology from his multi-fidelity branch which is coming in soon